pkg/crypto/keys/nep2_test.go

```diff
@@ -43,3 +43,39 @@ func TestNEP2Decrypt(t *testing.T) {
 		assert.Equal(t, testCase.Address, address)
 	}
 }
+
+func TestNEP2DecryptErrors(t *testing.T) {
+	p := "qwerty"
```

Comment:
nit: what about having random strings here and below?

Reply:
Well, they can randomly produce something that will successfully decrypt or not test the branch they're supposed to test, it's a bit dangerous and potentially unreproducible.